systest/cluster/nodes.go

```diff
@@ -183,8 +182,12 @@ func deployNodes(ctx *testcontext.Context, name string, from, to int, flags []De
 	)
 	for i := from; i < to; i++ {
 		i := i
+		finalFlags := make([]DeploymentFlag, len(flags)+1)
+		copy(finalFlags, flags)
 		eg.Go(func() error {
 			setname := fmt.Sprintf("%s-%d", name, i)
+			idx := i % ctx.PoetSize
+			finalFlags[len(finalFlags)-1] = PoetEndpoint(poetEndpoint(idx))
 			if err := deployNode(ctx, setname, labels, flags); err != nil {
```

Comment:
this should use finalFlags :)

Reply:
yes. i just realized that too. soooooooo dumb.....thanks for helping